Product/Production/Gateway/PatientDiscovery_10/src/main/resources/patientdiscovery/_10/webservices.xml

```diff
@@ -13,27 +13,33 @@
        xmlns:aop="http://www.springframework.org/schema/aop"
        xmlns:cxf="http://cxf.apache.org/core"
        xmlns:p="http://cxf.apache.org/policy"
+       xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="
         http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
         http://cxf.apache.org/bindings/soap         http://cxf.apache.org/schemas/configuration/soap.xsd
         http://cxf.apache.org/jaxws                 http://cxf.apache.org/schemas/jaxws.xsd
         http://cxf.apache.org/policy                http://cxf.apache.org/schemas/policy.xsd
-        http://cxf.apache.org/core                  http://cxf.apache.org/schemas/core.xsd">
+        http://cxf.apache.org/core                  http://cxf.apache.org/schemas/core.xsd
+        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
 
+    
     <!-- Patient Discovery -->
 
     <cxf:bus>
         <cxf:features>
             <p:policies />
         </cxf:features>
     </cxf:bus>
+    <context:property-placeholder />
```

Comment:
Just some left over cleanup, This was when we were trying to inject the jaxws properties into the endpoints. This is no longer needed and can be removed, along with the schemaLocations above.